external-import/wiz/src/external_import_connector/converter_to_stix.py

```diff
@@ -16,33 +13,24 @@ class ConverterToStix:
     def __init__(self, helper, config):
         self.helper = helper
         self.config = config
-        self.author = self.create_author(self.helper.connect_name)
-        self.external_reference = self.create_external_reference(
+        self.author = self._create_author(self.helper.connect_name)
```

Comment:
Since you need to work on the author, it would be a good idea to remove the `self.helper.connect_name` and replace it with `Wiz, Inc.`

```suggestion
        self.author = self._create_author("Wiz, Inc.")
```

Reply:
It'll be done in another issue